KPI-team1-frontend/src/pages/AddKpiModalPage.tsx

```diff
@@ -0,0 +1,456 @@
+import { useState } from "react";
+import ModalRightSide from "../components/ModalRightSide";
+import { IoCloseOutline } from "react-icons/io5";
+import {
+  Box,
+  FormControl,
+  FormControlLabel,
+  RadioGroup,
+  Radio,
+  Tooltip,
+} from "@mui/material";
+import { deepPurple } from "@mui/material/colors";
+import { HiMiniInformationCircle } from "react-icons/hi2";
+import { supabase } from "../supabase";
+
+export default function AddKpiModalPage({
+  isOpen,
+  onRequestClose,
+  circleId,
+  fetchKpiDefinitions,
+}: {
+  isOpen: boolean;
+  onRequestClose: () => void;
+  circleId: number;
+  fetchKpiDefinitions: () => Promise<void>;
+}): JSX.Element {
+  const [kpiName, setKpiName] = useState("");
+  const [description, setDescription] = useState("");
+  const [periodicityValue, setPeriodicityValue] = useState("daily");
+  const [unitValue, setUnitValue] = useState("numeric");
+  const [formulaValue, setFormulaValue] = useState("aggregate");
+  const [cumulativeValue, setCumulativeValue] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const addKpiToCircle = async (kpiId: number) => {
+    try {
+      const { data, error } = await supabase
+        .from("circle_kpi_definition")
+        .insert([
+          {
+            circle_id: circleId,
+            kpi_id: kpiId,
+          },
+        ])
+        .select("*");
+
+      if (error) {
+        console.log("Error adding new KPI to circle:", error.message);
+      }
+      if (data) {
+        fetchKpiDefinitions();
+      }
+    } catch (error: any) {
+      alert(error.message);
+    }
+  };
+
+  const addKpiToHistory = async (kpiId: number) => {
```

Comment:
We don't need this function anymore, the kpi will appear even if we don't create placeholders value

Reply:
But it doesn't work if I remove that function. The new KPI doesn't appear on "kpi_definition_with_latest_values" table 🤷🏻‍♀️